dot/core/service.go

```diff
@@ -550,3 +551,66 @@ func (s *Service) GetMetadata(bhash *common.Hash) ([]byte, error) {
 	rt.SetContextStorage(ts)
 	return rt.Metadata()
 }
+
+type (
```

Comment:
Can we declare this at the top or somewhere in `lib/common`?

Reply:
I move to the top